client/src/app/site/motions/components/motion-detail/motion-detail.component.ts

```diff
@@ -162,17 +196,10 @@ export class MotionDetailComponent extends BaseComponent implements OnInit {
                 this.router.navigate(['./motions/' + response.id]);
             });
         } else {
-            this.repo.save(newMotionValues, this.motionCopy).subscribe();
+            this.repo.update(newMotionValues, this.motionCopy).subscribe();
```

Comment:
`newMotionValues` is still of type object. Change if you want to enforce type safety during the update process.